README.md

```diff
@@ -20,15 +20,15 @@ The projects `CREDENTIAL_CONFIG.yml` file contains a list of variables which nee
 git clone https://github.com/<forked_repository_org>/tower_dummy_credentials
 ```
 
-2. Change the variable values in the `CREDENTIAL_CONFIG.yml` file. A list of all variables that need to be changed along with their usage can be found [HERE](VARIABLES.md).
+3. Make a copy of the `CREDENTIAL_CONFIG.yml` file and set the variables for your tower instance in the copied file `cp CREDENTIAL_CONFIG.yml my_credential_config.yml` . A list of all variables that need to be changed along with their usage can be found [HERE](VARIABLES.md).
```

Comment:
I'm starting to think that we should name `CREDENTIAL_CONFIG.yml` to `CREDENTIAL_CONFIG_TEMPLATE.yml` and then have users `cp` it to a file named `CREDENTIAL_CONFIG.yml`

```
cp CREDENTIAL_CONFIG_TEMPLATE.yml CREDENTIAL_CONFIG.yml
```